hoomd/hpmc/modules.h

```diff
@@ -15,17 +16,31 @@ namespace hpmc
 namespace detail
     {
 void export_sphere(pybind11::module& m);
+
 void export_convex_polygon(pybind11::module& m);
+
 void export_simple_polygon(pybind11::module& m);
+
+void export_simple_polygon_alchemy(pybind11::module& m);
+
 void export_spheropolygon(pybind11::module& m);
+
 void export_polyhedron(pybind11::module& m);
+
 void export_ellipsoid(pybind11::module& m);
+
 void export_faceted_ellipsoid(pybind11::module& m);
+
 void export_sphinx(pybind11::module& m);
+
 void export_union_convex_polyhedron(pybind11::module& m);
+
 void export_union_faceted_ellipsoid(pybind11::module& m);
+
 void export_union_sphere(pybind11::module& m);
+
 void export_convex_polyhedron(pybind11::module& m);
+
```

Comment:
```suggestion
void export_convex_polygon(pybind11::module& m);
void export_simple_polygon(pybind11::module& m);
void export_simple_polygon_alchemy(pybind11::module& m);
void export_spheropolygon(pybind11::module& m);
void export_polyhedron(pybind11::module& m);
void export_ellipsoid(pybind11::module& m);
void export_faceted_ellipsoid(pybind11::module& m);
void export_sphinx(pybind11::module& m);
void export_union_convex_polyhedron(pybind11::module& m);
void export_union_faceted_ellipsoid(pybind11::module& m);
void export_union_sphere(pybind11::module& m);
void export_convex_polyhedron(pybind11::module& m);
```
This change is not needed.